src/lib.rs

```diff
@@ -100,6 +102,13 @@ impl<K, V> LinkedHashMapEntry<K, V> {
     }
 }
 
+unsafe fn drop_empty_entry_box<K, V>(the_box: *mut LinkedHashMapEntry<K, V>) {
+    // Prevent compiler from trying to drop the un-initialized key and values in the node.
+    let LinkedHashMapEntry { key: k, value: v, .. } = *Box::from_raw(the_box);
```

Comment:
@apasel422 that's so cool! Changed!
I didn't about about this destructing variant.